apps/builder/app/canvas/shared/styles.ts

```diff
@@ -116,7 +116,43 @@ const subscribePreviewMode = () => {
 const subscribeEphemeralStyle = (params: Params) => {
   // track custom properties added on previous ephemeral styles change
   const addedCustomProperties = new Set<string>();
+
+  let timeoutHandle: number | undefined = undefined;
```

Comment:
isn't this called timerId or timeoutId?

Reply:
handle in programming is an abstract reference to a resource that allows programmers to manage and interact with that resource 
All my life it was handle, have no idea why js named that timerId